### ISSUE TITLE
Protonup doesn't catch my Kvantum theme on QT6

### ISSUE THREAD

**Issue by KaspianDev** [DavidoTek/ProtonUp-Qt#94]
**Describe the bug**  
I'm using KvLibadwaita theme with kvantum.
Everything works fine with qbitttorrent (which is a qt6 app)
However protonup is also qt6 app, yet my theme isn't found by the program

**To Reproduce**  
1. Install kvantum
2. Set kvantum as override with `QT_STYLE_OVERRIDE=kvantum-dark` in flatpak overrides
3. Launch protonup

**Expected behavior**  
Theme should be the same as for other qt6 apps.

**Screenshots**  
![image](https://user-images.githubusercontent.com/34658474/179797866-f837cad9-8540-42d9-9f3e-97450871a11c.png)

**Desktop (please complete the following information):**  
 - OS: Manjaro
 - Version 2.6.5
**Terminal output**  
```
ProtonUp-Qt 2.6.5 by DavidoTek. Build Info: DavidoTek Flathub build.
Python 3.9.9 (main, Nov 10 2011, 15:00:00) [GCC 11.3.0], PySide 6.3.1
Platform: Linux-5.15.55-1-MANJARO-x86_64-with-glibc2.33
QApplication: invalid style override 'kvantum-dark' passed, ignoring it.
	Available styles: Windows, Fusion
Loaded ctmod Proton-GE
Loaded ctmod Wine-GE
Loaded ctmod Boxtron
Loaded ctmod Kron4ek Wine-Builds Vanilla
Loaded ctmod Lutris-Wine
Loaded ctmod Luxtorpeda
Loaded ctmod Proton Tkg
Loaded ctmod Roberta
Loaded ctmod DXVK
Loaded ctmod DXVK Async
qt.pysideplugin: Environment variable PYSIDE_DESIGNER_PLUGINS is not set, bailing out.
qt.pysideplugin: No instance of QPyDesignerCustomWidgetCollection was found.
Gamepad error: No gamepad found.

```
edit: seems to work after restart?

**Comment by teppyboy** [comment]
Don't use Flatpak build, it will not use your system theme, instead use AUR package since you're using Manjaro.

**Comment by KaspianDev** [comment]
> Don't use Flatpak build, it will not use your system theme, instead use AUR package since you're using Manjaro.

all my other flatpaks use the theme i set (i have needed overrides), thats not a fix

**Comment by teppyboy** [comment]
> > Don't use Flatpak build, it will not use your system theme, instead use AUR package since you're using Manjaro.
> 
> all my other flatpaks use the theme i set (i have needed overrides), thats not a fix

Yeah, that isn't a fix but rather a workaround to your problem right now.

**Comment by KaspianDev** [comment]
> > > Don't use Flatpak build, it will not use your system theme, instead use AUR package since you're using Manjaro.
> > 
> > 
> > all my other flatpaks use the theme i set (i have needed overrides), thats not a fix
> 
> Yeah, that isn't a fix but rather a workaround to your problem right now.

Just wanna add that did not help sadly

**Comment by DavidoTek** [comment]
I think there have been some theming changes with Qt 6.4.
Does the problem still persist with the newest version?

**Comment by KaspianDev** [comment]
2.7.6 yep it still does

**Comment by DavidoTek** [comment]
Okay, thanks for the info.
